ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -196,8 +197,9 @@ class NetworkDropdown extends Component {
             {nickname || rpcUrl}
           </span>
           {isCurrentRpcTarget ? null : (
-            <i
-              className="fa fa-times delete"
+            <ButtonIcon
+              iconName={ICON_NAMES.CLOSE}
+              className="delete"
```

Comment:
Make sure you are updating the styles accordingly. From what I can see this would break because there is no `.fa` class. Could you also remove some of the styles in favour of `Box` props? `color`, `margin-right` etc

![Screenshot 2023-02-06 at 3 11 31 PM](https://user-images.githubusercontent.com/8112138/217108351-0a03e5d5-c7a6-499e-aa06-94cb830bbbed.png)